### ISSUE TITLE
[Feature request] Allow custom skip tones

### ISSUE THREAD

**Issue by erkinalp** [ajayyy/SponsorBlock#1064]

**Comment by ajayyy** [comment]
This might be difficult with browser extension storage limits, but maybe it could ask for a hosted url instead.

**Comment by erkinalp** [comment]
Why would extension storage limits matter? The default tone will be bundled as a separate file in the extension bundle, and external file would be provided by the user from the local filesystem.

**Comment by ajayyy** [comment]
I don't think it can just reference an external file, it would have to store it in the limited storage

**Comment by mchangrh** [comment]
> I don't think it can just reference an external file, it would have to store it in the limited storage

it would have to be accessed through `file://` or stored in localStorage
